purchase_batch_invoicing/data/ir_cron_data.xml

```diff
@@ -4,14 +4,14 @@
 
 <odoo>
 
-<record id="cron" model="ir.cron">
-    <field name="name">Invoice all pending purchase orders</field>
-    <field name="active" eval="False"/>
-    <field name="interval_type">weeks</field>
-    <field name="numbercall">-1</field>
-    <field name="model">purchase.batch_invoicing</field>
-    <field name="function">cron_invoice_all_pending</field>
-    <field name="args">("partner_id",)</field>
-</record>
+    <record id="cron" model="ir.cron">
+        <field name="name">Invoice all pending purchase orders</field>
+        <field name="active">True</field>
```

Comment:
Please don't change this field. Leave it disabled (and it has to be evaluated with `eval`).